src/tak/Game.java

```diff
@@ -208,6 +218,18 @@ void resign(Player p) {
         whenGameEnd();
     }
     
+    void setMarked(boolean marked, String field)
+    {
+        int file = field.charAt(0) - 'a';
+        int rank = field.charAt(1) - 1;
+        this.board[rank][file].setMarked(marked);
```

Comment:
I am working on it. For some reason I get a weird error in the NetBeans environment.